piksi_tools/serial_link.py

```diff
@@ -161,6 +176,74 @@ def log_printer(sbp_msg, **metadata):
   m = MsgLog(sbp_msg)
   print levels[m.level], m.text
 
+def swriter(link):
+  """Callback intended for reading out messages from one stream and into
+  a serial link stream.
+
+  Parameters
+  ----------
+  link : file handle
+
+  """
+  def scallback(sbp_msg, **metadata):
+    link(sbp_msg)
+  return scallback
```

Comment:
this looks kind of confusing reading here - had a hard time seeing that the return was with swriter, not with scallback.